src/main/java/org/jabref/gui/fieldeditors/FieldEditors.java

```diff
@@ -84,8 +84,10 @@ public static FieldEditorFX getForField(final Field field,
             } else {
                 return new OptionEditor<>(new TypeEditorViewModel(field, suggestionProvider, fieldCheckers));
             }
-        } else if (fieldProperties.contains(FieldProperty.SINGLE_ENTRY_LINK) || fieldProperties.contains(FieldProperty.MULTIPLE_ENTRY_LINK)) {
-            return new LinkedEntriesEditor(field, databaseContext, suggestionProvider, fieldCheckers);
+        } else if (fieldProperties.contains(FieldProperty.SINGLE_ENTRY_LINK)) {
+            return new LinkedEntriesEditor(field, databaseContext, suggestionProvider, fieldCheckers, false);
```

Comment:
Please do not Booleean parameters. Please use enums. See also [Java by Comparsion](https://java.by-comparison.com/): http://media.pragprog.com/titles/javacomp/split.pdf